src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java

```diff
@@ -1988,7 +1991,19 @@ public SBApiResponse profileMDOAdminUpdate(Map<String, Object> request, String u
 								Map<String, Object> updatedProfessionalDetailsMap = ((List<Map<String, Object>>) profileDetailsMap
 										.get(changedObj)).get(0);
 								for (String childKey : updatedProfessionalDetailsMap.keySet()) {
-									String updatedValue = (String) updatedProfessionalDetailsMap.get(childKey);
+									String updatedValue = null;
+									Object value = updatedProfessionalDetailsMap.get(childKey);
+									if (value != null) {
+										updatedValue = value.toString();
+									}
```

Comment:
you have introduced new checks but not completed on else parts.

is there a chance that updatedProfessionalDetailsMap.get(childKey) becomes null ?? how did our validation methods accepted it ?

if the value is null -- and we are simply going to accept it ?